index.html

```diff
@@ -11756,7 +11756,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 			<sdef>aria-hidden</sdef>
 			<div class="state-description">
 				<p><a>Indicates</a> whether the <a>element</a> is exposed to an accessibility API. See related <sref>aria-disabled</sref>.</p>
-				<p>User agents determine an element's [=element/hidden=] status based on whether it is rendered, and the rendering is usually controlled by CSS. For example, an element whose <code>display</code> property is set to <code>none</code> is not rendered. An element is considered [=element/hidden=] if it, or any of its ancestors are not rendered or have their <code>aria-hidden</code> attribute value set to <code>true</code>.</p>
+				<p>User agents determine an element's [=element/hidden=] status based on whether it is rendered, and the rendering is usually controlled by CSS. For example, an element whose <code>display</code> property is set to <code>none</code> is not rendered. An element will be <a href="#tree_exclusion">excluded from the accessibility tree</a> if it or any of its <a>owning elements</a> are [=element/hidden=] or have their <code>aria-hidden</code> attribute value set to <code>true</code>.</p>
```

Comment:
i think my only issue with this update is that the clarification about when to take aria-owns into account is so far removed from this initial mention of owning elements.  As prior to reading the new paragraph added on line 12333 of this diff, i would have interpreted the use of "owning elements" to mean that "boo" should also be "hidden" because the hidden div "owns" boo... but for the fact on 12333 it is stated that's not true because the owning element is itself not rendered, so thus it can't own the "boo" element.

```
<div hidden aria-owns=boo>foo</div>
<div id=boo>boo</div>
```
i could see this clarified by either mentioning that if an element is an owning element due to usage of aria-owns, how this works is clarified later.  Or that these bits of text are closer together in the spec, or that this paragraph remains just about ancestors being hidden, and owning elements / aria-owns headaches are talked about in one new paragraph/sets of paragraphs.

Reply:
Heya @scottaohara, now that the accessibility parent/child language has been merged in, I [updated this statement](https://github.com/w3c/aria/pull/1839/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R11607) to reference `accessibility parents` instead of `owning elements`. I think this is now more accurate and also sidesteps the ambiguity you described here, at least until an author digs deeper into the `aria-owns` section? Here’s the full new sentence:

> An element will be excluded from the accessibility tree if it or any of its **accessibility parents** are hidden or have their `aria-hidden` attribute value set to `true`.

@spectranaut, I’m seeing that this would be the spec’s first mention of _accessibility **parents**_ plural. I re-read that term’s definition and it feels okay to pluralize it here to essentially mean _ancestors_. Do you agree?

Reply:
Ahhh actually no, there is only one "accessibility parent" -- I think the appropriate phrase here is "Accessibility ancestor". We don't have it defined, but I think it is a clear enough term? You could link to the accessibility tree section of the document: https://w3c.github.io/aria/#accessibility_tree